test/android/BUILD.gn

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2016 Intel Corporation. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//build/config/android/rules.gni")
+
+# TODO(heke123): implement xwalk_core_unittests.
+
+shared_library("xwalk_core_unittests") {
```

Comment:
Sorry, this part is not finished yet.  In xwalk_android_test.gypi:
```
'target_name': 'xwalk_core_unittests_apk',
       'type': 'none',
       'dependencies': [
         'xwalk_core_unittests',
         'xwalk_core_unittests_java',
         'xwalk_core_unittests_jni',
       ],
```
I'll add this part as soon as possible, please ignore this temporarily.